.github/workflows/run-eval-pf-pipeline.yml

```
@@ -18,7 +23,8 @@ env:
   OPENAI_API_KEY: ${{ secrets.OPENAI_KEY }} 
   OPENAI_API_BASE: ${{ vars.OPENAI_ENDPOINT }}
   AISEARCH_KEY: ${{ secrets.AISEARCH_KEY }} 
-  AISEARCH_ENDPOINT: ${{ vars.AISEARCH_ENDPOINT }} 
+  AISEARCH_ENDPOINT: ${{ vars.AISEARCH_ENDPOINT }}
+  AISEARCH_INDEX: ${{ github.event.inputs.search_index || vars.AISEARCH_INDEX || 'patents' }} 
```

```suggestion
  AISEARCH_INDEX: ${{ github.event.inputs.search_index || 'patents' }} 
```

that variable does not seem necessary

---

I've removed the variable, and if we need to change the index we can change the default in the pipeline per branch for now. 👍